src/Timers.cpp

```diff
@@ -616,7 +640,7 @@ PVR_ERROR Timers::AddTimer(const kodi::addon::PVRTimer& timer)
       timer.GetClientIndex(),
       timer.GetClientChannelUid(),
       (int)timer.GetStartTime(),
-      (int)(timer.GetEndTime() - timer.GetStartTime()),
+      (int)(endTime - timer.GetStartTime()),
```

Comment:
C++ cast please

Might as well fix the line before this one while you are at it ;)

Reply:
There were a lot so I was trying to be consistent so I guess I must change them all.  Done